LibreOfficeWriter_DirectFormating.au3

```diff
@@ -10,23 +10,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Modifying and Applying Direct Character and Paragraph formating in L.O. Writer.
```

Comment:
Paragraph ?

Reply:
Yes, some direct formatting is applied specifically for Characters, others, such as Tab Stops,  etc, I believe are applied specifically to paragraphs. At least as best as I can tell.

Reply:
I was confused but now understand.
Thanks.